tinygrad/renderer/cstyle.py

```diff
@@ -339,12 +339,12 @@ def get_kernel_modifier(self, uops:List[UOp]) -> str:
     # https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html
     return f"__launch_bounds__({maxThreadsPerBlock}) "
 
-code_for_op_hip = { UnaryOps.SQRT: lambda x,dtype: f"__ocml_sqrt_f{ {dtypes.half:16, dtypes.double:64}.get(dtype, 32)}({x})",
-                    UnaryOps.SIN: lambda x,dtype: f"__ocml_sin_f{ {dtypes.half:16, dtypes.double:64}.get(dtype, 32)}({x})",
-                    UnaryOps.LOG2: lambda x,dtype: f"__ocml_log2_f{ {dtypes.half:16, dtypes.double:64}.get(dtype, 32)}({x})",
-                    UnaryOps.EXP2: lambda x,dtype: f"__ocml_exp2_f{ {dtypes.half:16, dtypes.double:64}.get(dtype, 32)}({x})",
+code_for_op_hip = { UnaryOps.SQRT: lambda x, dtype: f"__ocml_sqrt_f{ {dtypes.half:16, dtypes.double:64}.get(dtype, 32) }({x})",
```

Comment:
ugh, why does spacing change...

Reply:
Sorry, I tried changing `{dtypes.half:16, dtypes.double:64}.get(dtype, 32)}` for `dtype.itemsize*8` but when I realized why it was wrong I revert the changes and missed reverting spaces too

Reply:
And about the spacing is the same, I didn't really missed it, when I was reverting the changes I thought might as well leave the fix for spacing, I found it a little inconsistent across the codebase. But now I know that for a separate PR.

![image](https://github.com/user-attachments/assets/d69bf249-255c-4a31-a9c5-9f398ba818cb)